src/mmw/js/src/modeling/views.js

```diff
@@ -199,6 +258,8 @@ var ToolbarTabContentsView = Marionette.CollectionView.extend({
     childView: ToolbarTabContentView,
 
     onRender: function() {
+        // TODO: Move this logic to the template (active tab should
+        // be stored in model state somewhere).
```

Comment:
I think I made this comment already, but I can't find it. So here it is again: There's a attribute on project called `active-scenario-slug` that is used to set the active tab. I did it that way since it required less state. If it's more of a pain to have it there, we should add it to scenario, then remove `active-scenario-slug`.